### PR TITLE
BGDIINF_SB-1740: Uncommented delete asset on S3

### PR DESCRIPTION
**Pull request by rebert**

### CODE REVIEW

**Comment by rebert** [comment]
@ltshb : your code is still working :-)

Tested locally (minIO) via admin interface and via API. Does it need some e2e tests? or is this too risky?

**Comment by ltshb** [comment]
> 2e tests

E2E will be needed but only for dev and int

**Comment by ltshb** [comment]
I was referring to the whole code, there is still some commented code in unittest and in models that needs to be un-commented and/or updated and/or removed. Search for BGDIINF_SB-1625 in the whole project.

**Comment by rebert** [comment]
@ltshb I did not remove BGDIINF_SB-1625 all over, as not related to this ticket. This is just about deleting an asset on s3 as well

**Comment by ltshb** [comment]
@rebert there is in the model a remove() function that probably can be removed and there is still some unittest for removing asset on S3 that are commented out, that is what I am referring to.

File: app/tests/test_assets_endpoint.py [hunk 1]
```diff
@@ -659,15 +660,20 @@ def setUp(self):  # pylint: disable=invalid-name
...
         self.maxDiff = None  # pylint: disable=invalid-name
 
+    @mock_s3_asset_file
     def test_asset_endpoint_delete_asset(self):
         collection_name = self.collection.name
         item_name = self.item.name
         asset_name = self.asset.name
         path = f'/{STAC_BASE_V}/collections/{collection_name}/items/{item_name}/assets/{asset_name}'
+        # Check that the file exists on s3
+        self.assertS3ObjectNotExists(path)
```
Comment (ltshb) [inline app/tests/test_assets_endpoint.py:670]: Here you probably want `assertS3ObjectExists` and not `assertS3ObjectNotExists` ;-)

**Review by ltshb — APPROVED** [review]

**Pull request merged**